sql/src/test/java/io/druid/sql/calcite/planner/CalcitesTest.java

```diff
@@ -42,14 +42,14 @@ public void testEscapeStringLiteral()
   @Test
   public void testFindOutputNamePrefix()
```

Comment:
nit rename test method name to testFindUnusedPrefix.

Reply:
Fixed that and re-pushed.

Reply:
Thanks @gianm